e2e-tests/content-strings/fields/index.js

```diff
@@ -188,6 +188,12 @@ export const FIELDS = {
     LABEL: 'Your UK telephone number (optional)',
     HINT: 'We may need to contact you about your application',
   },
+  [FIELD_IDS.INSURANCE.CURRENCY.ALTERNATIVE_CURRENCY_CODE]: {
+    TEXT: 'Another currency',
+  },
+  [FIELD_IDS.INSURANCE.POLICY.CONTRACT_POLICY.ALTERNATIVE_POLICY_CURRENCY_CODE]: {
+    TEXT: 'Another currency',
+  },
```

Comment:
Duplicate `TEXT` here, is this intentional?

Reply:
yes.  We have 2 different fieldIds for same field.  Policy one and then a generic one